pkgs/development/ocaml-modules/kicadsch/default.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, buildDunePackage
+, fetchFromGitHub
+, ocaml
+}:
+
+assert (lib.versionAtLeast ocaml.version "4.09");
```

```suggestion
assert (lib.versionAtLeast ocaml.version "4.09");
```
Why not use `minimalOCamlVersion` in buildDunePackage?